packages/batch/src/processPartialResponseSync.ts

```diff
@@ -41,17 +41,37 @@ import type {
  *     context,
  *   });
  * ```
```

Comment:
Let's add a line in between these two examples that says something like: "When working with SQS FIFO queues, we will stop processing at the first failure and mark unprocessed messages as failed to preserve ordering. However, if you want to continue processing messages from different group IDs, you can enable the `skipGroupOnError` option for seamless processing of messages from various group IDs.`

Also let's leave new lines between the example and the text.